replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+
+clean_1b_low_resort <- clean_1b %>%
+  filter(split == 0, resort == 1) %>%
+  mutate(mean.beer = mean(beer), CI = (1.96 * Sem(beer))) %>%
+  distinct(mean.beer, CI)
+#generates mean and CI for low income participants in the resort scenario - to use in figure 1b
+```
+
+> and beer from the store.
+
+```{r echo = FALSE}
+clean_1b %>%
+  filter(split == 0, resort == 0) %>%
+  summarise(mean.beer = mean(beer), CI.low = mean(beer) -  (1.96 * Sem(beer)), CI.high = mean(beer) + (1.96 * Sem(beer))) %>%
+  round(digits = 2)
+#1.96 * standard error gives 95% CI in one direction
+#this calculates low income WTP beer price at grocery store
+
+clean_1b_low_grocer <- clean_1b %>%
+  filter(split == 0, resort == 0) %>%
+  mutate(mean.beer = mean(beer), CI = (1.96 * Sem(beer))) %>%
+  distinct(mean.beer, CI)
+#generates mean and CI for low income participants in the grocer scenario - to use in figure 1b
+```
+
+
+> The interaction between income and context was significant F(1,145) = 11.18, p < 0.01, eta squared partial = 0.07.
+
+```{r echo = FALSE}
+
+summary(aov_1b <- aov(beer ~ resort * split, data = clean_1b))
+
+temp_eta <- etaSquared(aov(beer ~ resort * split, data = clean_1b)) %>%
+  .[3,2] %>%
+  round(digit = 2)
+print(c("partial eta squared:", temp_eta), quote = FALSE)
+
+```
+
+> The interaction was also significant when income was treated as a continuous variable beta = 2.27, t(145) = 2.17, p < 0.05.
+
+```{r echo = FALSE}
+lm(scale(beer) ~ scale(resort) + scale(logses) + scale(resort * logses), data = clean_1b) %>%
+  tidy() %>%
+  filter(term == "scale(resort * logses)") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4))  %>%  
+        select(term, beta, p)
+
+```
+
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
+values = c(clean_1b_low_grocer$mean.beer, clean_1b_high_grocer$mean.beer, clean_1b_low_resort$mean.beer, clean_1b_high_resort$mean.beer)
+location = c("Grocer", "Grocer", "Resort", "Resort")
+CIs = c(clean_1b_low_grocer$CI, clean_1b_high_grocer$CI, clean_1b_low_resort$CI, clean_1b_high_resort$CI)
+fig1b.table <- data.frame(title, values, location, CIs)
+#figure 1b data frame
+
+figure1b <- fig1b.table %>%
+  ggplot(aes(x = title, y = values, fill = location)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("WTP (dollars)") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + theme_bw() + scale_fill_grey(start = .3, end = .85) + ggtitle("Figure 1b - Amount participants were willing to pay (WTP)\nfor beer at the grocery store and resort in Study 1b\n(error bars represent 95% confidence interval)") + scale_y_continuous(breaks = seq(0, 8, 1)) + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 1b
```

Comment:
I second this. Your code at times is quite dense which makes it harder to follow.